App.tsx

```diff
@@ -28,8 +28,15 @@ import AddTodo from 'components/Todo/AddTodo';
 const store = configureStore();
 sagaMiddleWare.run(rootSaga);
 
+export type ModalStackParamsList = {
+  Main: undefined;
```

Comment:
Why `undefined` in type?

Reply:
https://reactnavigation.org/docs/typescript#type-checking-the-navigator

Typecheck for params.

Reply:
will check